### ISSUE TITLE
Revert my shit

### ISSUE THREAD

**Issue by Swartz27** [xrOxygen/xray-oxygen#148]
I fucked up somewhere, do not use this currently. I'm trying to figure out what I did wrong.

**Comment by ForserX** [comment]
Okay.

**Comment by Swartz27** [comment]
https://www.youtube.com/watch?v=jP5e2XJsB_4  <-- Me on this strain XD  Back to Sativa, I hate Indica, especially the memory loss...

**Comment by ForserX** [comment]
```cpp
    D3D11_FEATURE_DATA_D3D11_OPTIONS2 features_2;
	HRESULT dxResultF2 = pDevice->CheckFeatureSupport(D3D11_FEATURE_D3D11_OPTIONS2, &features_2, sizeof(features_2));
	if (dxResultF2 == S_OK)
	{
		features_2.ConservativeRasterizationTier >= D3D11_CONSERVATIVE_RASTERIZATION_TIER_1;
	}
```
Fixed version. 
```cpp
features_2.ConservativeRasterizationTier >= D3D11_CONSERVATIVE_RASTERIZATION_TIER_1;
```
Dead code.

**Comment by Swartz27** [comment]
Thanks, however that is not the only issue (I fucked up some shaders: this can be fixed by doing a search in the shader files and deleting all instances of "uniform float4 screen_res" that are not in common.h. There may be other mistakes as well. Luckily I found a way to sober up, so now I can take a proper look.

**Comment by ForserX** [comment]
I'm fixed all problems. Check: https://github.com/xrOxygen/xray-oxygen/commit/73c48f4e95a4ff055167327c83aa91759f38536e

Closed #137
